ui/app/styles/components/diff-version-selector.scss

```diff
@@ -0,0 +1,24 @@
+.visual-diff {
+  background-color: black;
+
+  pre {
+    color: $ui-gray-010;
+  }
+}
+
```

Comment:
over-riding the packages colors. We wanted a specific red and green to match the reds and greens we use in the app. confirmed colors with design.